app/controllers/search_controller.rb

```diff
@@ -2,4 +2,19 @@ class SearchController < ApplicationController
   def index
     @gyms = BackEndFacade.get_gyms_near_user(params['zip_code'])
   end
+
+  def show
+    @gym = BackEndFacade.get_selected_gym(params[:id])
```

Comment:
I saw your note in the PR - I exposed the following route to return users at the same gym that the user is NOT already friends with:

`/users/{user_id}/gym_members?yelp_gym_id={yelp_gym_id}`

This should feed the bottom section of gym show page wireframe (section to add friends)

Reply:
For the top section (current friends at the same gym), the existing `/users/{user_id}/friendships` route was refactored to take query param, so without `yelp_gym_id`, you get all user friends (for dashboard), and with query param, you get friends at same gym:

```ruby
  def index
    user = User.find(params[:user_id])
    friends = if params[:yelp_gym_id]
                user.friends_at_same_gym(params[:yelp_gym_id])
              else
                user.followees
              end
     ...
```

Reply:
Please let me know if I misinterpreted your question or if you need additional BE support for the required data